recipes/implot/all/conanfile.py

```diff
@@ -60,7 +60,12 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["implot"]
-        
+
         self.cpp_info.set_property("cmake_file_name", "implot")
         self.cpp_info.set_property("cmake_target_name", "implot")
```

Comment:
Again, we can avoid future breakage by removing `self.cpp_info.set_property("cmake_target_name", "implot")` (and add `self.cpp_info.set_property("cmake_target_name", "implot::implot")` later).